### DISCUSSION TITLE
Improper installation location for 64-bit version on Windows?

### DISCUSSION THREAD

**Discussion by mikunimaru** [google/mozc#893] [Q&A]
**Description**
Even though I installed Mozc with the 64-bit installer, Mozc is placed in the installation location for 32-bit.

**Steps to reproduce**
1. Click on Mozc64.msi to install Mozc.

**Expected behavior**
Mozc will be installed under 'Program Files'.

**Actual behavior**
Mozc will be installed under 'Program Files (x86)'.

**Version or commit-id**
Mozc-2.28.5080.100+24.11.oss

**Environment**
 - OS:Windows

**Comment by yukawa** [comment]
Thank you for your suggestion.

This is actually a known issue inherited from Google Japanese Input, and for now we'd like to keep the current configuration to minimize the diff between Google Japanese Input and OSS Mozc.

The thing is that the following executables are still 32-bit even if you install `Mozc64.msi`.

* `mozc_tool.exe`
* `mozc_ja_tip32.dll`
* `msvcp140.dll`
* `vcruntime140.dll`
* `Qt5Core.dll`
* `Qt5Gui.dll`
* `Qt5Widgets.dll`
* `platforms/qwindows.dll`

So for those binaries, the location `%ProgramFiles(x86)%/Mozc` is indeed consistent.

I know it looks a bit strange, but given that there is no user visible impact I'd like to keep the current installation location until Google Japanese Input also starts using new installation location, which may require careful migration strategy for existing users.
